src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Implements Get-ExperimentalFeature cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "ExperimentalFeature", HelpUri = "")]
+    public class GetExperimentalFeatureCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Specify the feature names.
+        /// </summary>
+        [Parameter(ValueFromPipeline = true, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        public string[] Name { get; set; }
+
+        /// <summary>
+        /// Search module paths to find all available experimental features.
+        /// </summary>
+        [Parameter()]
```

Comment:
Parentheses are used here but not above in a no-arg attribute -- should probably be consistent

Reply:
Fixed.